sdk/communication/Azure.Communication.MediaComposition/src/autorest.md

```diff
@@ -7,11 +7,19 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 title: Media Composition
-tag: package-2022-07-16-preview1
-generation1-convenience-client: true
+tag: package-2022-07-16-preview
 model-namespace: false
+generation1-convenience-client: true
 payload-flattening-threshold: 10
-require:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e2de3bd42679ab281a61c52b4a24252d5394449a/specification/communication/data-plane/MediaComposition/readme.md
+input-file:
+    - mediaComposition.json
+```
 
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions.*
+  transform: >
+    $["x-csharp-usage"] = "model,input,output";
+    $["x-csharp-formats"] = "json";
```

Comment:
interesting, what's the reason that we need these transforms and what are they doing?

Reply:
We had some issues with all the models generating as `internal`. I started a thread on the autorest channel and this was what I found. I couldn't find a lot of information on what the transforms do but they seem to be overriding the accessibility.
https://teams.microsoft.com/l/message/19:acbd512e57bd475198ea6bf4564599e3@thread.skype/1658189263863?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1658189263863&teamName=Azure%20SDK&channelName=AutoRest%20-%20CSharp&createdTime=1658189263863&allowXTenantAccess=false

Reply:
Ah, I remember that thread. Not an intuitive fix but a fix nonetheless.